mitmproxy/tools/console/flowlist.py

```diff
@@ -57,22 +59,27 @@ def get_focus(self):
         return f, self.master.view.focus.index
 
     def set_focus(self, index):
-        if self.master.commands.execute("view.properties.inbounds %d" % index):
+        if self.master.view.inbounds(index):
             self.master.view.focus.index = index
 
-    def get_next(self, pos):
-        pos = pos + 1
-        if not self.master.commands.execute("view.properties.inbounds %d" % pos):
+    def __get(self, pos):
+        if not self.master.view.inbounds(pos):
             return None, None
         f = FlowItem(self.master, self.master.view[pos])
         return f, pos
 
+    def __get_cached(self, pos):
```

Comment:
How about we just use `functools.lru_cache(maxsize=None)` for `__get` and then call `self.__get.cache_clear()` in `view_changed`?